.github/workflows/ci.yml

```diff
@@ -135,7 +135,7 @@ jobs:
             !presto-hive,!presto-orc,!presto-parquet,
             !presto-mongodb,!presto-kafka,!presto-elasticsearch,
             !presto-redis,
-            !presto-sqlserver,!presto-postgresql,!presto-mysql,
+            !presto-sqlserver,!presto-postgresql,!presto-mysql,!presto-memsql,
```

Comment:
Move this to be after `phoenix`. This list should be in the same order as the one below.